lets-encrypt/activate-tls.sh

```diff
@@ -220,44 +220,34 @@ if [ -n "$DEDYNDOMAIN" ]
 then
     print_text_in_color "$ICyan" "Renewing TLS with DNS, please don't abort the hook, it may take a while..."
     # Renew with DNS by default
-    certbot --manual \
-    --text \
-    --rsa-key-size 4096 \
-    --renew-by-default \
-    --server https://acme-v02.api.letsencrypt.org/directory \
-    --no-eff-email \
-    --agree-tos \
-    --preferred-challenges dns \
-    --manual-auth-hook "$SCRIPTS"/deSEC/hook.sh  \
-    --manual-cleanup-hook "$SCRIPTS"/deSEC/hook.sh \
-    -d "$DEDYNDOMAIN" \
-    certonly
-else
-    generate_cert "$TLSDOMAIN"
-fi
-
-# Generate DHparams
-if [ -d "$CERTFILES" ]
-then
-    if [ ! -f "$DHPARAMS_TLS" ]
+    if certbot certonly --manual --text --rsa-key-size 4096 --renew-by-default --server https://acme-v02.api.letsencrypt.org/directory no-eff-email --agree-tos --preferred-challenges dns --manual-auth-hook "$SCRIPTS"/deSEC/hook.sh --manual-cleanup-hook "$SCRIPTS"/deSEC/hook.sh -d "$DEDYNDOMAIN"
```

Comment:
Why did you put all of this into one line? It is much harder to read like this, imho...

Reply:
to do the `if` statement.

Reply:
But this is also possible with using many lines. See for example: https://github.com/nextcloud/vm/blob/277908e21a2b48891c5e4a5f7c357c85e32b914c/not-supported/borgbackup.sh#L369-L370

Reply:
ping @enoch85

Reply:
Feel free to create a PR where that's sorted.